### PR TITLE
Resolve2, IdentifyLite working with team id or `tid:teamid` assertion

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits c4c7f03340f8...01c99b5d9566 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496860463559-c4c7f03340f8-01c99b5d9566-linux/Tracker-New_user_Scroll_1-c4c7f03340f8.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496860463559-c4c7f03340f8-01c99b5d9566-linux/Tracker-New_user_Scroll_1-01c99b5d9566.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496860463559-c4c7f03340f8-01c99b5d9566-linux/Markdown-emoji-c4c7f03340f8.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496860463559-c4c7f03340f8-01c99b5d9566-linux/Markdown-emoji-01c99b5d9566.png)  
   Markdown-emoji comparison
</summary>

File: go/libkb/assertion.go [hunk 1]
```diff
@@ -404,7 +404,7 @@ func (a AssertionUID) CheckAndNormalize(_ AssertionContext) (AssertionURL, error
 }
 
 func (a AssertionTeamID) ToLookup() (key, value string, err error) {
-	return "tid", a.Value, nil
+	return "id", a.Value, nil
```
Comment (maxtaco) [inline go/libkb/assertion.go]: Why was this the right idea?

Comment (patrickxb) [inline go/libkb/assertion.go]: That's the parameter to "lookup", i.e. to `team/get`.

**Comment by keybase-ci-visdiff** [comment]
The commits e8b9c572a393...cd7ad173a3e0 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496868678711-e8b9c572a393-cd7ad173a3e0-linux/Tracker-New_user_Scroll_1-e8b9c572a393.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496868678711-e8b9c572a393-cd7ad173a3e0-linux/Tracker-New_user_Scroll_1-cd7ad173a3e0.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496868678711-e8b9c572a393-cd7ad173a3e0-linux/Markdown-emoji-e8b9c572a393.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496868678711-e8b9c572a393-cd7ad173a3e0-linux/Markdown-emoji-cd7ad173a3e0.png)  
   Markdown-emoji comparison
</summary>

**Comment by patrickxb** [comment]
@strib this requires kbfs changes.  How should we coordinate?

**Comment by strib** [comment]
@patrickxb: Should we make it `Resolve3` so it doesn't require coordination?  I worry, especially on Linux, about the service getting upgraded before the running KBFS, and this breaking everything.

**Comment by patrickxb** [comment]
Sounds good.  I'll change it.

**Comment by keybase-ci-visdiff** [comment]
The commits 2552d4879294...876001770aa7 introduce visual changes on linux. <details><summary>:mag_right: 2 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496939447836-2552d4879294-876001770aa7-linux/Tracker-New_user_Scroll_1-2552d4879294.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496939447836-2552d4879294-876001770aa7-linux/Tracker-New_user_Scroll_1-876001770aa7.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **Markdown-emoji** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496939447836-2552d4879294-876001770aa7-linux/Markdown-emoji-2552d4879294.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1496939447836-2552d4879294-876001770aa7-linux/Markdown-emoji-876001770aa7.png)  
   Markdown-emoji comparison
</summary>

**Pull request merged**